src/Nri/Ui/Tooltip/V3.elm

```diff
@@ -1223,115 +1221,103 @@ positionTooltip direction alignment =
         ltrPosition =
             case alignment of
                 Start customOffset ->
-                    Css.left customOffset
+                    [ Css.left customOffset, Css.right Css.unset ]
```

Comment:
This change is related to [this change](https://github.com/NoRedInk/noredink-ui/pull/1659/files#diff-69b280c9b764b0a83df1dbec0ed407a0cfab0f5b56346abec0bd7c2c19030e2eR1051-R1083), which was made because `withViewport` was dropped.

As far as I could tell, that function is explicitly only being used as a workaround for the broken cascading. With that fixed, we can generate CSS for only the breakpoints we need, but we need to make sure that other properties up the cascade (that we aren't already overriding) are `unset`.

I'd prefer to not bring `withViewport` back, but if we didn't want to make these changes as drastic we could add another combinator to MediaQuery like `MediaQuery.narrowMobile |> MediaQuery.and MediaQuery.not MediaQuery.mobile` to target a specific viewport in a more generalized way.